tests/generators/test_jenkins/testGitOptions.xml

```diff
@@ -20,7 +20,6 @@
       <reference>/home/reference.git</reference>
       <timeout>30</timeout>
     </hudson.plugins.git.extensions.impl.CloneOption>
-    <hudson.plugins.git.extensions.impl.CleanCheckout/>
```

Comment:
In `testGItOptions`, `clean_checkout` as `false` but the tag was still being written. This is now fixed.